packages/webapp/src/utils/credits-purchase-constants.js

```diff
@@ -1,12 +1,20 @@
 const routes = {
-  ESTIMATOR_CREDITS_INDIVIDUAL_ORG: '/credits-purchase/applying-individual-organisation'
+  APPLYING_INDIVIDIAL_ORGANISATION: '/credits-purchase/applying-individual-organisation',
+  CREDITS_PURCHASE_CHECK_DEFRA_ACCOUNT_DETAILS: '/credits-purchase/check-defra-account-details'
+  // REGISTER_CREDIT_PURCHASE_TASK_LIST: '/credit-purchase/register-credit-purchase-task-list'
```

Comment:
This can be uncommented if minimal task list support is added in this pull request (see earlier comment).

Reply:
No longer required in this pull request due to inclusion in https://github.com/DEFRA/biodiversity-net-gain-service/pull/615.